g/glassfish/glassfish_ubi_9.3.sh

```diff
@@ -0,0 +1,74 @@
+
+# Install Java 
+wget https://github.com/adoptium/temurin17-binaries/releases/download/jdk-17.0.6%2B10/OpenJDK17U-jdk_ppc64le_linux_hotspot_17.0.6_10.tar.gz
+tar -C /usr/local -xzf OpenJDK17U-jdk_ppc64le_linux_hotspot_17.0.6_10.tar.gz
+export JAVA_HOME=/usr/local/jdk-17.0.6+10
+export JAVA17_HOME=/usr/local/jdk-17.0.6+10
+export PATH=$PATH:/usr/local/jdk-17.0.6+10/bin
+ln -sf /usr/local/jdk-17.0.6+10/bin/java /usr/bin
+rm -f OpenJDK17U-jdk_ppc64le_linux_hotspot_17.0.6_10.tar.gz
+java -version
+
+# Install maven
+wget https://dlcdn.apache.org/maven/maven-3/3.9.8/binaries/apache-maven-3.9.8-bin.tar.gz
+tar -xvzf apache-maven-3.9.8-bin.tar.gz
+cp -R apache-maven-3.9.8 /usr/local
+ln -s /usr/local/apache-maven-3.9.8/bin/mvn /usr/bin/mvn
+rm -f apache-maven-3.9.8-bin.tar.gz
+mvn -version
+
+# Set MAVEN_OPTS environment variable
+export MAVEN_OPTS="-Xmx2500m -Xss768k -XX:+UseG1GC -XX:+UseStringDeduplication"
+
+# Clone package repository
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! mvn -B -e clean install -Pfastest,staging -T4C; then
+        echo "------------------$PACKAGE_NAME:build_fails-------------------------------------"
+        echo "$PACKAGE_VERSION $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Build_Fails"
+	exit 1
+fi
+
+# Several modules cause test failure, excluding those modules during testing
+if ! mvn test -pl '!nucleus/osgi-platforms/felix, !nucleus/grizzly/nucleus-grizzly-all, !nucleus/common/common-util, !nucleus/core/bootstrap, !nucleus/deployment/common, !nucleus/deployment/dtds, !nucleus/deployment/schemas, !nucleus/distributions/nucleus-common, !nucleus/distributions/atomic, !nucleus/distributions/nucleus, !appserver/deployment/dtds, !appserver/deployment/schemas, !appserver/admin/template, !appserver/ejb/ejb-timer-databases, !appserver/connectors/descriptors, !appserver/jms/jmsra, !nucleus/glassfish-jul-extension, !appserver/jdbc/jdbc-ra/jdbc-ra-distribution, !appserver/persistence/cmp/cmp-scripts, !appserver/batch/batch-database, !appserver/extras/jakartaee/dist-frag, !appserver/extras/appserv-rt/dist-frag, !appserver/grizzly/glassfish-grizzly-extra-all, !appserver/webservices/webservices-scripts, !appserver/appclient/client/appclient-scripts, !appserver/webservices/metro-fragments, !appserver/distributions/glassfish-common, !appserver/distributions/glassfish, !appserver/extras/embedded/common/bootstrap, !appserver/extras/embedded/shell/glassfish-embedded-shell-frag, !appserver/extras/embedded/shell/glassfish-embedded-static-shell-frag, !appserver/extras/embedded/all, !appserver/extras/embedded/web, !appserver/distributions/web, !appserver/tests/admin, !appserver/tests/admin/tests, !appserver/tests/appserv-tests/lib, !appserver/tests/appserv-tests, !appserver/tests/appserv-tests/connectors-ra-redeploy/rars, !appserver/tests/appserv-tests/connectors-ra-redeploy/rars-xa, !appserver/tests/embedded/scatteredarchive, !docs/distribution'; then
```

Comment:
@ramnathnayak-ibm Are these failing for both intel and power?

Reply:
Yes